seed-build.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+function cleanup() {
+  for job in $( jobs -p ); do
+    kill -SIGTERM "${job}"
+    wait "${job}"
+  done
+}
+trap cleanup EXIT
+
+echo "[INFO] Starting the development server..."
+rm -rf dev.db
+cp dev.db.initialized dev.db
+set +o errexit
+pnpm dev &
+set -o errexit
+for (( i = 0; i < 10; i++ )); do
+  if ! curl -s http://localhost:3000; then
+    echo "Dev server not yet started, waiting..."
+    sleep 1
+  else
+    break
+  fi
+done
+
+echo "[INFO] Logging in with the bootstrap user and seeding the database..."
+curl -s -H 'Content-Type: application/json' -H 'Accept: application/json' -X POST http://localhost:3000/api/users/login --data '{"email":"bootstrap@avy.com","password":"test"}' > login.json
```

Comment:
What I did to generate `dev.db.initialized` was just start the server and create the user with these credentials. Unfortunately, if we can't automate *that*, we have the same problem because changes to the schemas without updated to the `dev.db.initialized` will cause hangups:

```
[✓] Pulling schema from database...
? Warnings detected during schema push: 

· You're about to add not-null fake column without default value, which contains 1 items

DATA LOSS WARNING: Possible data loss detected if schema is pushed.

Accept warnings and push schema to database? › (y/N)
```

Reply:
Let me see if I can automate the first user as well ...

Reply:
Yeah, already done, sorry